packages/docusaurus-plugin-ideal-image/src/index.ts

```diff
@@ -35,12 +35,12 @@ export default function (
               use: [
                 require.resolve('@docusaurus/lqip-loader'),
                 {
-                  loader: require.resolve('@endiliey/responsive-loader'),
+                  loader: require.resolve('responsive-loader-modern'),
```

Comment:
I don't think it will really ever need an update again, so what is the point

Reply:
For security and documentation reasons, I'll republish your fork as 1.4.0 (it's 1.3.2 on npm currently which may be misleading in the future, particularly when you explain Webpack 5 support is since 1.4.0 🤪)